src/cordova/app.cordova.ts

```diff
@@ -27,7 +27,7 @@ function onDeviceReady() {
   initializeQRReader()
   initializeClipboard(cordova)
 
-  document.addEventListener("backbutton", onBackKeyDown, false)
+  document.addEventListener("backbutton", event => contentWindow.postMessage({ id: "backbutton" }, "*"), false)
```

Comment:
Let's try to replicate the actual `backbutton` event on the iframe's `document`. Timeboxed to ~15 mins.

Reply:
Screw it, it works fine and we need that feature 😉

Aiming for a release today, so this gotta be merged.